packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/WorkingGroupLeadOpening/CreateWorkingGroupLeadOpening/ApplicationForm.tsx

```diff
@@ -6,16 +6,22 @@ import { Row } from '@/common/components/Modal'
 import { RowGapBlock } from '@/common/components/page/PageContent'
 import { TextMedium } from '@/common/components/typography'
 
-export const ApplicationForm = () => {
+interface Props {
+  noHeader?: boolean
+}
+
+export const ApplicationForm = ({ noHeader = false }: Props) => {
```

Comment:
Lets keep the components simple, instead of adding a `noHeader` prop please extract everything after the header to a new component in `@/working-groups/components/CreateOpening/ApplicationForm.tsx`
Same thing for `@/working-groups/components/CreateOpening/DurationAndProcess.tsx`, `@/working-groups/components/CreateOpening/StakingPolicyAndReward.tsx`, and `@/working-groups/components/CreateOpening/WorkingGroupAndDescription.tsx`.

This way the `proposals` domain imports from `working-groups` instead of the opposite which in this case is IMO more in agreement with the [documentation](https://github.com/Joystream/pioneer/tree/dev/packages/ui#directory-structure).